service/application/src/main/resources/application.yml

```diff
@@ -36,7 +36,7 @@ spring:
     allow-bean-definition-overriding: true
 
 camel:
-  springboot:
+  main:
```

Comment:
Was this an intentional change?
We would need to test that the MDC attributes are still working for a scan initiated from Solace cloud (we carry state through MDC like the traceId and actorId that needs to be present and eventually sent back to Solace Cloud in the response).

Reply:
Yeah, I saw that it's deprecated so I switched it over. I didn't test it thoroughly, so I will remove it and get that sorted out on ep-core's side and EMA's side another time.